commit b6bc765067ece933cc3dc7f5e95665a89100b1d5
Author: Joe Perches <joe@perches.com>
Date:   Tue Dec 21 02:16:08 2010 -0800

    drivers/net/*.c: Use static const
    
    Using static const generally increases object text and decreases data size.
    It also generally decreases overall object size.
    
    Signed-off-by: Joe Perches <joe@perches.com>

--- a/drivers/net/r8169.c
+++ b/drivers/net/r8169.c
@@ -1862,7 +1862,7 @@ static void rtl8168d_2_hw_phy_config(voi
 
 		val = mdio_read(ioaddr, 0x0d);
 		if ((val & 0x00ff) != 0x006c) {
-			u32 set[] = {
+			static const u32 set[] = {
 				0x0065, 0x0066, 0x0067, 0x0068,
 				0x0069, 0x006a, 0x006b, 0x006c
 			};
